js/index.js

```diff
@@ -1,19 +1,20 @@
 const keys = document.querySelectorAll('.key')
 const note = document.querySelector('.current-key')
 const hints = document.querySelectorAll('.key__hints')
-
+let repeatedKeys = false
```

Comment:
```suggestion

let repeatedKeys = false

```

This is not critical, but it is important to maintain code readability and follow styleguides